mqtt/mqtt-broker/src/lib.rs

```diff
@@ -183,3 +192,12 @@ pub enum Message {
     Client(ClientId, ClientEvent),
     System(SystemEvent),
 }
+
+#[cfg(test)]
+pub(crate) mod tests {
```

Comment:
is this needed?

Reply:
nope, it is rust_analyzer generated snippet.